blog/_posts/2015-08-16-Introducing-Team-Alpha-Ruby.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Introducing Team Alpha Ruby (α ❤ ✌)
+layout: post
+created_at: Sun Aug 16 2015
+permalink: blog/2015-07-30-introducing-alpha-ruby
+current: blog
+author: Ariane and Ramya
+twitter: teamalpharuby
+---
+
+Hello everybody! We are [Team Alpha Ruby](https://teams.railsgirlssummerofcode.org/teams/83), Ariane and Ramya - no team name pun intended ;-).
+
+During Rails Girls Summer of Code 2015, we will be working on the [Official Ruby Documentation Redesign](http://docs.ruby-lang.org/)
+
+Our team has wonderful support on this journey from our mentor Zachory Scott and our coaches Barbara Barbosa and Stella Miranda.
+Furthermore Benedikt Deicke provides additional input and help as our team’s supervisor.
+
+<img src="/img/blog/2015/introducing-team-alpha-ruby.jpg" alt="Team Alpha Ruby" width="600">  
+<font color="grey"><small><i>The amazing members of Team Alpha Ruby</i></small></font> 
```

Comment:
We just need an image credit here (like who took the picture/who the image belongs to). See other teams' posts if you're having trouble, but usually the credit was given to the team or to a single team member. :)